sonar-db/src/main/java/org/sonar/db/qualityprofile/QualityProfileDto.java

```diff
@@ -123,6 +124,16 @@ public QualityProfileDto setLastUsed(@Nullable Long lastUsed) {
     return this;
   }
 
+  @CheckForNull
+  public Long getUserUpdatedAt() {
+    return userUpdatedAt;
+  }
+
+  public QualityProfileDto setUserUpdatedAt(@Nullable Long userUpdatedAt) {
```

Comment:
Forgot this question. when no modification has been made by a user on a profile, this column will be null